docs/administration/dashboard/php_api_dashboard_service.md

```diff
@@ -0,0 +1,8 @@
+---
+title: PHP API Dashboard service
+description: PHP API Dashboard service
+---
+
+# PHP API Dashboard service
```

Comment:
We should either expand this section (it's only one sentence at the moment) or remove it.

Reply:
It is draft PR, still in progress, section will be expanded.